docs/podman-create.1.md

```diff
@@ -804,13 +804,28 @@ KillMode=process
 WantedBy=multi-user.target
 ```
 
+### Rootless Containers
+
+Podman runs as a non root user on most system. This feature requires that a new enough version of shadow-utils
+be installed.  The shadow-utils package must include the newuidmap and newgidmap executables.
+
+Note: RHEL7 and Centos 7 will not have this feature until RHEL7.7 is released.
+
+In order for a user to run rootless, the must be an entry for the username in /etc/subuid and /etc/subgid
+listing the UIDs for his usernamespace.
+
+Rootless podman works better if the fuse-overlayfs and slirp4netns packages are installed.
+fuse-overlay provides a userspace overlay storage driver, otherwise you need to use the vfs storage driver, which
```

Comment:
This is not wrong, but personal preference.  I don't like starting a sentence with a lower case name like this.  I'd prefer "The fuse-overlay package", ditto next line.

Reply:
Fixed